db/migrate/20160311050505_create_events.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
+      t.datetime :start_at
+      t.datetime :end_at
+      t.string :location
+      t.integer :user_id
+      t.integer :tutor_id
```

Comment:
if you add the `tutor_id` and the `user_id` just as integers then you will manually have to create the relationship between the two models at the database level.
A work around for this is to use the generator helper methods: `rails g model Event tutor:belongs_to user:has_many`
This will automatically set up the relation in the database and set a index on the `tutor_id` and the `user_id` columns for better search and filter.